### PR TITLE
Add Users feature

### PR DESCRIPTION
**Pull request by recurator**
Add Users feature
Sign up/Sign in using devise.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+
+  gem 'rspec-rails'
+
+  gem 'cucumber-rails', require: false
+
+  gem 'database_cleaner'
+
+  gem 'webrat'
```
Comment (Ikera) [inline Gemfile]: Do you actually use `webrat` for any tests? As I can notice from Github repo https://github.com/brynary/webrat last 5 - 7 years nobody maintains that gem.

File: features/users.feature [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        Then I should see "Invalid Email or password"
+
+      Scenario: User is registered and enters correct email and password
+        Given a registered user with the email "jabre@example.net" with password "secret" exists
+        And I am on the "/users/sign_in" page
+        Then I should see "Log in"
+        When I fill in "user_email" with "jabre@example.net"
+        And I fill in "user_password" with "secret"
+        And I press "Log in"
+        And I should see "Signed in successfully"
```
Comment (Ikera) [inline features/users.feature]: Indentation needs to be fixed.

File: Gemfile (hunk 1 above)
Comment (recurator) [reply Gemfile]: fixed

Comment (Ikera) [reply Gemfile]: Did you push your commit with fixes to Github? I cannot see the new commit.

File: features/users.feature (hunk 2 above)
Comment (recurator) [reply features/users.feature]: fixed

**Comment by Ikera** [comment]
@recurator 👍

**Pull request merged**